users/fmzakari/secrets/secrets.nix

```diff
@@ -0,0 +1,12 @@
+let
+  systems = {
+    nyx = "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOXmzOre8wnaZm4zXuXqzFRS+5GFlMyfhth9ie9AvW8t root@nyx";
```

Comment:
@markrwilliams this is how I could control if i waned some secrets to be decryptable by your machine.